content/components/selectpanel.mdx

```diff
@@ -125,24 +433,64 @@ Select panel should be anchored to its trigger.
       width="100%"
       src="https://user-images.githubusercontent.com/40274682/228390156-dab0b413-3b90-4736-a0e7-8d5d30665c43.mp4"
     />
-    <Caption>Pressing esc, or selecting “cancel” should close the current dialog and enter focus to the previous trigger. It should not retain or submit any user input.</Caption>
+    <Caption>
+      Pressing esc, or selecting “cancel” should close the current dialog and enter focus to the previous trigger. It
+      should not retain or submit any user input.
+    </Caption>
   </div>
   <div>
     <CustomVideoPlayer
       width="100%"
       src="https://user-images.githubusercontent.com/40274682/228390297-d6d59818-1d1e-47bb-ac9d-cd1eccc57ba8.mp4"
     />
-    <Caption>Selecting the submit button or pressing the enter key while focused in the listbox should submit any user input.</Caption>
+    <Caption>
+      Selecting the submit button or pressing the enter key while focused in the listbox should submit any user input.
+    </Caption>
   </div>
 </Box>
 
 ### Responsive
 
-Select panel can become a full screen dialog or a bottom sheet at narrow viewports. Support at 320px is required if the select panel is larger at wider viewports. See [narrow-viewport placement](https://primer.style/design/components/dialog#narrow-viewport-placement) in the Dialog guidelines for more details.
+On smaller viewports, the select panel can be transformed into either a full-screen dialog or a bottom sheet dialog. To ensure usability on touch devices, the confirmation buttons and action list items are also increased to a medium size, ensuring a sufficiently large hit target.
+
+When adding an extra action in the footer, it is important to ensure that it still fits within a 320px viewport.
+
+<img
+  width="960"
+  alt="mobile"
+  src="https://github.com/primer/design/assets/980622/ae6d19c2-7bfa-4357-be65-341c0728b9a8"
+/>
+
+See the [Dialog guidelines](https://primer.style/design/components/dialog) for more details.
+
+## Cancel/submit
+
+In most scenarios, we recommend using declarative controls, except for some single-select instances as [demonstrated earlier](#list-single-select).
+However, we acknowledge that in specific areas of the product that users interact with frequently throughout the day, employing declarative controls may create the perception of a sluggish experience.
+For example, if the issue creation process involves too many steps, it may discourage customers from reporting issues.
```

Comment:
I really don't agree with this statement here. The issues people had with it taking too many steps doesn't deal with the fact that people have to confirm their choices, which hasn't yet been implemented. The issues they encountered was due to poor UI. 

I want to dispel the notion that choosing to confirm selections from a dialog "adds another step", and would prefer to not have that in our official documentation.

Reply:
Removing this section to push this pull request through.